README.md

```diff
@@ -1,6 +1,43 @@
 # VIP Go / WordPress.com Compatibility
 
-This plugin provides compatibility for sites that are moving from WordPress.com to VIP Go.
+This plugin provides compatibility for sites that are moving from WordPress.com to VIP Go. It's meant to be a temporary shim to ease your transition to the new platform, not a permanent solution.
+
+_We suggest reading through this README and the codebase to determine whether you need it or it can be removed safely._
+
+## Plugin Versioning
+
+To minimize the amount of moving pieces we move plugins with exactly the same version as on WordPress.com. With the exception of `es-wp-query` because versions prior to `0.2.0` are not compatible with VIP Go.
+
+`wpcom_vip_load_plugin` has a couple of important differences on VIP Go:
+
+1. `wpcom_vip_load_plugin` does not accept `$version` argument.
+2. Plugins can only be loaded from `plugins/` and its subfolders.
+
+To address those differences we automatically move all loaded plugins during the code migration into `plugins`. If we detect that `wpcom_vip_load_plugin` call uses $version argument we replace that call with `wpcom_vip_legacy_load_plugin`.
+
+`wpcom_vip_legacy_load_plugin` is a thin wrapper around `wpcom_vip_load_plugin` that allows for loading of the correct version.
+
+⚠️ If you see calls to `wpcom_vip_legacy_load_plugin` in your codebase (themes or your custom plugins) removing `vip-go-wpcom-compat` will break your site. ⚠️
+
+Examples:
+
+#### Unversioned
+```
+// WP.com loads from `vip/plugins/my-plugin`
+wpcom_vip_load_plugin( 'my-plugin' );
+
+// VIP Go loads from `plugins/`
+wpcom_vip_load_plugin( 'my-plugin' );
+```
+
+#### Versioned
+```
+// WP.com loads from `vip/plugins/my-plugin-2.0`
+wpcom_vip_load_plugin( 'my-plugin', 'plugins', '2.0' );
+
+// VIP Go loads from `plugins/`
+wpcom_vip_load_plugin( 'my-plugin' );
```

Comment:
Should this include a version to show how that gets handled on Go?

Reply:
Yes, yes it should :) Thanks for the catch!